src/libraries/Microsoft.Extensions.Caching.Memory/tests/CapacityTests.cs

```diff
@@ -341,6 +341,23 @@ public async Task ExpiringEntryDecreasesCacheSize()
             Assert.Equal(0, cache.Size);
         }
 
+        [Fact]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/36037")]
```

Comment:
Please remove `ActiveIssue`.  This attribute is used to disable a test against an issue when the test is failing.   You don't need that here.
```suggestion
```

Reply:
Hi Eric, thanks for your review. The attribute has been removed. Additionally I have added a test for an expired token.